ui/appui-layout-react/src/appui-layout-react/base/NineZoneState.ts

```diff
@@ -1090,9 +1093,11 @@ export function addPopoutWidget(state: NineZoneState, id: PopoutWidgetState["id"
 
 /** @internal */
 export function addTab(state: NineZoneState, id: TabState["id"], tabArgs?: Partial<TabState>): NineZoneState {
+  const hideWithUiWhenFloating = tabArgs?.hideWithUiWhenFloating ? tabArgs.hideWithUiWhenFloating : false;
```

Comment:
Similar to `createTabState`, spread of `tabArgs` should be enough here?

Reply:
When I was writing this, I had to add this explicitly to get it to work properly. That could have been a symptom of something else broken, though, so I'll try without the explicit prop.